app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,5 @@
-/*
- * Copyright 2011-2018, The Trustees of Indiana University and Northwestern
+/* 
```

Comment:
Odd that the scss files are getting a space at the end of their first lines.  Probably not worth dealing with though.